odoo/addons/base/tests/test_configmanager.py

```diff
@@ -0,0 +1,274 @@
+            'from_filter': '.*',
+            'smtp_server': 'localhoost',
+            'smtp_port': 12,
+            'smtp_ssl': True,
+            'smtp_user': 'spongebob',
+            'smtp_password': 'Tigrou007',
+            'smtp_ssl_certificate_filename': '/var/www/cert',
+            'smtp_ssl_private_key_filename': '/var/www/key',
+
+            # database
+            'db_name': 'horizon',
+            'db_user': 'kiwi',
+            'db_password': 'Tigrou007',
+            'pg_path': '/binary/pg_path',
+            'db_host': True,
+            'db_port': 4269,
+            'db_sslmode': 'verify-full',
+            'db_maxconn': 42,
+            'db_maxconn_gevent': True,
+            'db_template': 'backup1706',
+            'db_replica_host': '192.168.0.255',
+            'db_replica_port': 2038,
+
+            # i18n
+            'load_language': 'fr_FR',  # blacklist for save, read from the config file
+            'language': 'fr_FR',  # blacklist for save, read from the config file
+            'translate_out': '/foo/bar/translate_out.csv',  # blacklist for save, read from the config file
+            'translate_in': '/foo/bar/translate_in.csv',  # blacklist for save, read from the config file
+            'overwrite_existing_translations': True,  # blacklist for save, read from the config file
+            'translate_modules': ['all'],  # ignored from the config file
+
+            # security
+            'list_db': False,
+
+            # advanced
+            'dev_mode': [],  # blacklist for save, ignored from the config file
+            'shell_interface': 'ipython',  # blacklist for save, read from the config file
+            'stop_after_init': True,  # blacklist for save, read from the config file
+            'osv_memory_count_limit': 71,
+            'transient_age_limit': 4.0,
+            'max_cron_threads': 4,
+            'unaccent': True,
+            'geoip_city_db': '/foo/bar/city.db',
+            'geoip_country_db': '/foo/bar/country.db',
+        }
+
+        if IS_POSIX:
+            # multiprocessing
+            values.update(
+                {
+                    'workers': 92,
+                    'limit_memory_soft': 1234,
+                    'limit_memory_hard': 5678,
+                    'limit_time_cpu': 3,
+                    'limit_time_real': 4,
+                    'limit_time_real_cron': -3,
+                    'limit_request': 1,
+                }
+            )
+
+        config_path = file_path('base/tests/config/non_default.conf')
+        config = configmanager(fname=config_path)
+        self.assertEqual(config.rcfile, config_path, "Config file path doesn't match")
+
+        config._parse_config()
+        self.assertEqual(config.options, values, "Options don't match")
+        self.assertEqual(config.rcfile, config_path)
+        self.assertNotEqual(config.rcfile, config['config'])  # funny
+
+    @unittest.skipIf(not IS_POSIX, 'this test is POSIX only')
```

Comment:
because of the inclusion of the multiprocessing options in the `save_proxis.conf` file